Source/JavaScriptCore/inspector/remote/glib/RemoteInspectorUtils.cpp

```diff
@@ -42,22 +42,21 @@ GRefPtr<GBytes> backendCommands()
 #if PLATFORM(WPE)
     static std::once_flag flag;
     std::call_once(flag, [] {
-        const char* libDir = PKGLIBDIR;
+        const char* dataDir = PKGDATADIR;
+        GError* error = nullptr;
```

Comment:
You could use a `GUniqueOutPtr` smart pointer here, and then the `g_clear_error()` call below is not needed.

```suggestion
        GUniqueOutPtr<GError> error;
```